### ISSUE TITLE
Error after overwrite sonata.user.form.registration.type

### ISSUE THREAD

**Issue by bartosz-malec** [sonata-project/SonataUserBundle#779]
I got an error after overwrite sonata.user.form.registration.type service:

An exception has been thrown during the rendering of a template ("Warning: Missing argument 1 for Sonata\UserBundle\Form\Type\RegistrationFormType::__construct(), called in C:\Work\fotbox\vendor\sonata-project\core-bundle\Form\Extension\DependencyInjectionExtension.php on line 94 and defined") in FOSUserBundle:Security:login.html.twig at line 14.

My service:

```
sonata.user.registration.form.type:
    class: Application\UserBundle\Form\Type\RegistrationFormType
    tags:
        - { name: form.type, alias: sonata_user_registration }
    arguments:
        - "%fos_user.model.user.class%"
        - "%sonata.user.registration.form.options%"
        - "@request_stack"
        - "@service_container"
```

Any solutions?

**Comment by Soullivaneuh** [comment]
Please first, fill GitHub issue template for bug report.

**Comment by OskarStark** [comment]
Closed via #869
